Source/Tests/HarnessHelpers.h

```diff
@@ -228,7 +228,7 @@ namespace FEX::HarnessHelper {
             Matches &= A == B;
           };
 
-          for (unsigned j = 0; j < Offset.second; ++j) {
+          for (unsigned j = 0; j < RegData->RegDataCount; ++j) {
```

Comment:
This depends on the test config file correctly setting this field, right?

Reply:
This is already getting set correctly in the python config parser.
RegCount always counts the number of 64bit registers the test is testing for.
We don't yet support smaller than 64bit here.